picker_wrapper.go

```diff
@@ -165,6 +165,14 @@ func (bp *pickerWrapper) pick(ctx context.Context, failfast bool, opts balancer.
 			}
 			return t, done, nil
 		}
+		if done != nil {
+			done(balancer.DoneInfo{
+				Err:           nil,
```

Comment:
These are all the defaults - maybe delete and a comment "default DoneInfo indicates RPC did not occur"?

Reply:
It's tricky to test this because returning a non-Ready SubConn from picker is very racy.